src/benchmarks/micro/libraries/System.Text.Json/Document/Perf.DocumentParse.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.IO;
+using System.Text.Json.Tests;
+
+namespace System.Text.Json.Document.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.JSON)]
+    public class Perf_DocumentParse
+    {
+        public enum TestCaseType
+        {
+            HelloWorld,
+            BasicJson,
+            Json400B,
+            Json400KB
+        }
+
+        [ParamsAllValues]
+        public TestCaseType TestCase;
+
+        [Params(true, false)]
+        public bool IsDataCompact;
```

Comment:
would `Indented` be a better and more consistent name?

Reply:
Maybe. `Indented` is the opposite of `IsDataCompact`, so we would have to flip the boolean if we  choose to change it. I think IsDataCompact is reasonably self-describing.

How is `Indented` more consistent?

Reply:
> How is Indented more consistent?

In the setup we have the following usage of this property: `new JsonWriterOptions { Indented = !IsDataCompact }`

So we have two names that describe the same thing

Reply:
I see what you mean.

That setup is for a different test (`Perf.ParseThenWrite.cs`) and using `Indented` is a little strange for a parsing-only test like `Perf.DocumentParse.cs` .

The way I thought about it was:
- IsDataCompact is about the characteristic of the data provided by the benchmark setup which being **read/parsed** to build a DOM.
- The Indented option is to control how to **write** the JSON back in the benchmark itself.

But you do bring up a good point about terminology. Maybe, rename it to `IsDataIndented` to avoid having multiple terms for the same concept?

```suggestion
        public bool IsDataIndented;
```